frontend/src/app/main/ui/ds/controls/combobox.scss

```diff
@@ -0,0 +1,87 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@use "../_borders.scss" as *;
+@use "../_sizes.scss" as *;
+@use "../typography.scss" as *;
+
+.combobox-wrapper {
+  --combobox-icon-fg-color: var(--color-foreground-secondary);
+  --combobox-fg-color: var(--color-foreground-primary);
+  --combobox-bg-color: var(--color-background-tertiary);
+  --combobox-outline-color: none;
+  --combobox-border-color: none;
+
+  &:hover {
```

Comment:
Place optional code (i.e. when on hover, when selected) after not optional code.